frontend/src/App.tsx

```diff
@@ -27,30 +22,16 @@ const App = (): React.ReactElement => {
     [authenticatedUser],
   );
 
-  // Some sort of global state. Context API replaces redux.
-  // Split related states into different contexts as necessary.
-  // Split dispatcher and state into separate contexts as necessary.
-  const [sampleContext, dispatchSampleContextUpdate] = useReducer(
-    sampleContextReducer,
-    DEFAULT_SAMPLE_CONTEXT,
-  );
-
   // Providers for app-specific state like contexts are here.
   // For providers for libraries like Apollo and OAuth, see index.tsx.
   return (
-    <SampleContext.Provider value={sampleContext}>
-      <SampleContextDispatcherContext.Provider
-        value={dispatchSampleContextUpdate}
-      >
-        <AuthContext.Provider value={currentAuthContext}>
-          <BrowserRouter>
-            <Header />
-            <Routes />
-            <Footer />
-          </BrowserRouter>
-        </AuthContext.Provider>
-      </SampleContextDispatcherContext.Provider>
-    </SampleContext.Provider>
+    <AuthContext.Provider value={currentAuthContext}>
+      <BrowserRouter>
+        <Header />
+        <Routes />
+        <Footer />
+      </BrowserRouter>
+    </AuthContext.Provider>
```

Comment:
Not exactly sure what is happening here, just to make sure, are you switching to FCK specific authentication instead of sample code?

Reply:
I just removed the sample context since it wasn't doing anything. The FCK specific authentication is still the same as before! ( I just deleted
```
 <SampleContext.Provider value={sampleContext}>
      <SampleContextDispatcherContext.Provider
        value={dispatchSampleContextUpdate}
      >
```)